config/enabletypedirectives/services.yaml

```diff
@@ -6,4 +6,4 @@ metadata:
 # spec:
 #   status:
 #     kind: FederatedServiceStatus
-#   enableStatus: true
```

Comment:
Please remove this TODO instead of maintaining it.

Reply:
Removed